README.md

```diff
@@ -60,8 +60,8 @@ Also see the [developer's guide](./docs/devguide.md) for information on how to
 build and test the code.
 
 We have weekly meetings - see
-[Kubernetes SIGs](https://github.com/kubernetes/community/blob/master/sig-list.md)
-(search for "Service Catalog") for the exact date and time. For meeting agendas
+[our SIG Readme](https://github.com/kubernetes/community/blob/master/sig-service-catalog/README.md#meetings)
+for details. For meeting agendas
 and notes, see [Kubernetes SIG Service Catalog Agenda](https://docs.google.com/document/d/17xlpkoEbPR5M6P5VDzNx17q6-IPFxKyebEekCGYiIKM/edit).
```

Comment:
comment while we're editing here. we have about a year of meetings in our current doc, should we consider rotating to a new doc?

Reply:
Does having a large doc cause any problems?

Reply:
gdocs doesn't load large docs well sometimes, and based on the device you're on.
Now that I'm back at my computer I'll move stuff....

Reply:
👍 Thanks!